django_forms_encoder/templatetags/django_forms_encoder.py

```diff
@@ -0,0 +1,12 @@
+"""Encoder template tags."""
+from django import template
+
+from django_forms_encoder.encoder import form_to_dict
+
+register = template.Library()
+
+
+@register.filter
+def encode_form(value):
+    """Convert django form to simple dictionary."""
+    return form_to_dict(value)
```

Comment:
`form_to_dict` could be registered directly as a template filter. Or a test should be added.

Reply:
Fixed.